apluslms_roman/configuration.py

```diff
@@ -56,16 +65,67 @@ def find_from(cls, path, name=None, prefixes=None):
                     break
         else:
             if is_legacy_configuration(path):
-                container = cls.Container(config, allow_missing=True)
+                container = cls.Container(join(path, cls.DEFAULT_FILENAME), allow_missing=True)
                 return cls(container, None, LEGACY_CONFIG, cls.version)
-            raise CourseConfigError("Couldn't find course configuration from {}. Expected to find one of these: {}".format(path, ', '.join(files)))
+            raise ProjectConfigError((
+                "Couldn't find course configuration from {}. Expected to find one of these: {}"
+            ).format(path, ', '.join(files)))
 
         return cls.load(config)
 
+    @classmethod
+    def load_from(cls, config):
+        if isfile(config):
+            return cls.load(config)
+        if '.' not in basename(config):
+            for prefix in cls.DEFAULT_PREFIXES:
+                filename = '%s.%s' % (config, prefix)
+                if isfile(filename):
+                    return cls.load(filename)
+        raise ProjectConfigError("Given file '{}' doesn't exist.".format(config))
+
+    @classmethod
+    def create(cls, filename):
+        document = cls.load(filename, allow_missing=True)
+        document.save(overwrite=False)
+        return document
+
+    def initialize_data(self):
+        self.recursive_update(
+            LEGACY_CONFIG if is_legacy_configuration(dirname(self.path))
+            else DEFAULT_CONFIG
+        )
+        self._data[self._version_key] = str(self.version)
```

Comment:
`super().inirialize_data()` ;)

Reply:
This line should not be necessary as the `super().initialize_data()` should set `self[self.version_key]`, if it there is version_key.